example/local_server.dart

```diff
@@ -0,0 +1,42 @@
+import "package:wampproto/serializers.dart";
```

Comment:
local_server is the wrong name. There is no server in this case. So this is in-memory-router